README.md

```diff
@@ -5,9 +5,9 @@ Small repo done in cab ride
 A simple repo used to track progress of codechef monthly contests from JAN15-JUNE17 (Can be increased or decresed by modifying code)
 
 ## What to do?
-* Install virtual env
+* Install virtual env 
 * Create an environment using virtual env `virtualenv envName -p python3`
-* Activate the environment and install the requirements `source bin/activate; pip3 install -r req.txt`
+* Activate the environment and install the requirements `source envName/bin/activate; pip3 install -r req.txt`
```

Comment:
you got me there. yep sorry project is kept inside the virtual env. so i think we should change the readme accordingly right? (a rough sketch)
->install virtual env
->create virtual env
->cd virtual env and then activate the virtual env. 
->clone the project inside the folder and cd to that folder.
->run req.txt and helper.py
->activate the server.